jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/ConfigMapEnricher.java

```diff
@@ -124,12 +150,13 @@ private void addConfigMapFromXmlConfigurations(KubernetesListBuilder builder) {
                         if (name == null) {
                             name = Paths.get(file).getFileName().toString();
                         }
-                        addConfigMapEntryFromFile(configMapBuilder, name, file);
+                        addConfigMapEntryFromDirOrFile(configMapBuilder, name, file);
                     }
                 }
             }
 
-            if ((configMapBuilder.getData() != null && !configMapBuilder.getData().isEmpty()) || (configMapBuilder.getBinaryData() != null && !configMapBuilder.getBinaryData().isEmpty())) {
+            if (configMapBuilder.getData() != null && !configMapBuilder.getData().isEmpty()
```

Comment:
Hi, thx for contributing and sorry for the belayed review, we were focused on 1.0.x release.

Why did we remove the parenthesis? Precedence is not that clear now.

Reply:
Hi, it's a my IDE option that remove unecessary parenthesis. I can remove it if you want

Reply:
Hi, yes please, leave it as it was. Intention looks clearer, I reviewed this fairly early in the morning and it took a while to figure out what was happening.

Reply:
Parenthesis added

Reply:
Thank you :pray: